clippy_lints/src/no_mangle_with_rust_abi.rs

```diff
@@ -0,0 +1,60 @@
+use clippy_utils::diagnostics::span_lint_and_sugg;
+use clippy_utils::source::snippet_with_applicability;
+use rustc_errors::Applicability;
+use rustc_hir::{Item, ItemKind};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_session::{declare_lint_pass, declare_tool_lint};
+use rustc_target::spec::abi::Abi;
+
+declare_clippy_lint! {
```

Comment:
Does it make sense to move this to `src/attrs.rs` or let it remain in it's own file?

Reply:
Feel free to leave in its own file.